cmd/openshift-install/log.go

```diff
@@ -56,6 +57,20 @@ func (h *fileHook) Fire(entry *logrus.Entry) error {
 		msgs = strings.Split(orig, "\n")
 	}
 
+	level := logrus.InfoLevel
+	if h.config.Level != nil {
+		parsedLevel, err := logrus.ParseLevel(*h.config.Level)
+		if err == nil {
+			level = parsedLevel
+		}
+	}
+
+	if h.config.Fields != nil && level <= entry.Level {
```

Comment:
Is the `level <= entry.Level` bit so we don't bother decorating if we're not going to emit the message? If so, why not just `return nil`?

Reply:
>Is the level <= entry.Level bit so we don't bother decorating if we're not going to emit the message? 
It's so we can control the level from the log-config. 

> If so, why not just return nil
This hook does other things in the installer and I am adding this functionality to it so, in general, I don't want to return nil.